dev/devicelab/lib/framework/utils.dart

```diff
@@ -529,19 +529,43 @@ String extractCloudAuthTokenArg(List<String> rawArgs) {
   return token;
 }
 
+final RegExp obsRegExp =
```

Comment:
These can be library private e.g. '_obsRegExp'

Reply:
Done.